lib/parse-mvn.js

```diff
@@ -4,30 +4,39 @@ var packageFormatVersion = 'mvn:0.0.1';
 var newline = /[\r\n]+/g;
 var logLabel = /^\[\w+\]\s*/gm;
 var digraph = /digraph([\s\S]*?)\}/g;
+var errorLabel = /^\[ERROR\]/gm;
 
 // Parse the output from 'mvn dependency:tree -DoutputType=dot'
 function parseTree(text, withDev) {
-  text = text.replace(logLabel, '');
   try {
-    return {
-      ok: true,
-      data: getRootProject(text, withDev),
-    };
+    // check for errors in mvn output
+    if (errorLabel.test(text)) {
+      throw new Error('Error: failed to execute an `mvn` command.');
+    }
```

Comment:
My thinking is this - if the output of the `mvn` command contains errors, we should not even attempt to parse the dependency tree.

Is this a valid assumption in the maven world?

Reply:
Asking because this is a change in behaviour of the plugin.